.travis.yml

```diff
@@ -15,9 +15,11 @@ install:
   - npm ci # faster, goes only from package-lock
 before_script:
   - psql -c 'create database "boilermaker-test";' -U postgres # remember to change this name if you change it elsewhere (e.g. package.json)
+script:
+  - npm test             # test the code
+  - npm run build-client # make the bundle
```

Comment:
Note that it is recommended to put everything which _might_ be considered a code authoring failure (as opposed to CI setup error) into `script`.

- This results in a difference between failure and error state icons e.g. next to commits in GitHub.
- Also, if one of these lines fails, it doesn't exit early, but rather continues to run other lines. In this way, if both the tests and build fail, the author finds out immediately rather than upon fixing _just_ the test or _just_ the build.